app/main.py

```diff
@@ -118,6 +115,16 @@
 alert_metadata = build_live_alerts_metadata()
 alert_id = alert_metadata["id"]
 
+# We define the coordinates of the point on which to center the map, as well as the appropriate zoom level for each
+# group of users (which will be used in the manage_login_modal callback below)
+group_correspondences = {
```

Comment:
We will need to hardcode this ? This dictionnary is never stored anywhere else ?

Reply:
If I understood well, what we will have for the tests in Ardèche on the API side is a way to fetch the `group_id` but there will not be fields like the center latitude and longitude and the zoom associated with it in the tables. So we would need to hardcode it, at least in a first version of the code (and as it will be very short, I guess it is simpler to hardcode it rather than storing it as a JSON file). But I might be wrong for the API and I am open to moving it to a static file if you prefer!

Reply:
Will do in the afternoon!